scripts/globals/conquest.lua

```diff
@@ -211,6 +211,82 @@ function tradeConquestGuard(player,npc,trade,guardnation,guardtype)
 
 end;
 
+function updateConquestGuard(player,csid,option,size,inventory)
+    if (option >= 32768 and option <= 32944) then
+        for Item = 1,size,3 do
+            if (option == inventory[Item]) then
+                local CPVerify = 1;
+                if (player:getCP() >= inventory[Item + 1]) then
+                    CPVerify = 0;
+                end;
+
+                player:updateEvent(2,CPVerify,inventory[Item + 2]);
+                break;
+            end
+        end
+    end
+end;
+
+function finishConquestGuard(player,csid,option,size,inventory,guardnation)
+    local CONQUEST = GetTextIDVariable(player:getZoneID(), "CONQUEST");
+    local ITEM_OBTAINED = GetTextIDVariable(player:getZoneID(), "ITEM_OBTAINED");
+    local ITEM_CANNOT_BE_OBTAINED = GetTextIDVariable(player:getZoneID(), "ITEM_CANNOT_BE_OBTAINED");
```

Comment:
Instead of making up new bindings, just variable-ize the zones textIDs file so its always "whatever zone we are in now".
```lua
        local TextIDs = "scripts/zones/" .. player:getZoneName() .. "/TextIDs";
        package.loaded[TextIDs] = nil;
        require(TextIDs);
```